config.sample.ini

```diff
@@ -16,20 +16,22 @@ vardir = /var/lib/puppetdb
 # disable-update-checking = true
 
 [database]
-# For the embedded DB: org.hsqldb.jdbcDriver
+
+# Note that support for the embedded DB (HSQLDB) has been deprecated
+# and will be removed in a future version:
+# https://groups.google.com/d/msg/puppet-users/8K5sPqNgErM/8PI5pjI5iRgJ
+
 # For PostgreSQL: org.postgresql.Driver
-# Defaults to embedded DB
-classname = org.hsqldb.jdbcDriver
+# For the embedded DB: org.hsqldb.jdbcDriver
+#classname = org.postgresql.Driver
 
-# For the embedded DB: hsqldb
 # For PostgreSQL: postgresql
-# Defaults to embedded DB
-subprotocol = hsqldb
+# For the embedded DB: hsqldb
+#subprotocol = postgresql
 
-# For the embedded DB: file:/path/to/database;hsqldb.tx=mvcc;sql.syntax_pgs=true
 # For PostgreSQL: //host:port/databaseName
-# Defaults to embedded DB located in <vardir>/db
-subname = file:/var/lib/puppetdb/db/db;hsqldb.tx=mvcc;sql.syntax_pgs=true
+# For the embedded DB: file:/path/to/database;hsqldb.tx=mvcc;sql.syntax_pgs=true
+#subname = //localhost:5234/puppetdb
```

Comment:
5432 is the PG default. Are we commenting the postgres examples so that the hsqldb defaults still work? If not I think it'd be fine to uncomment them despite them requiring some config.